.github/workflows/checks.yml

```diff
@@ -206,6 +171,14 @@ jobs:
         run: |
           poetry install --without py39-dev
 
+      # ====================== SETUP DEVNET ====================== #
+
+      - name: Install devnet
+        run: |
+          cargo install --locked \
+          --git https://github.com/0xSpaceShard/starknet-devnet-rs.git \
+          --rev ${{ env.DEVNET_SHA }}
```

Comment:
It may be a good idea to set `--root` to better control where the devnet is installed

Reply:
I was initially doing it this way, but given that it works very well without `--root` I would leave it as it is. If we want to change it at any point, it would be a quick fix.

Reply:
sure, this is a nitpick, but please remember that this will mean that if someone modifies cwds at any point in the future, the tests may break